src/status_im2/contexts/shell/activity_center/notification/reply/view.cljs

```diff
@@ -14,16 +14,23 @@
 
 ;; NOTE: Replies support text, image and stickers only.
 (defn- get-message-content
-  [{:keys [content-type] :as message}]
+  [{:keys [content-type] :as message} albumMessages]
   (case content-type
     constants/content-type-text [quo/text {:style style/tag-text}
                                  (get-in message [:content :text])]
 
     constants/content-type-image
-    (let [image           (get-in message [:content :image])
-          image-local-url (url/replace-port image (rf/sub [:mediaserver/port]))
-          photos          (when image-local-url [{:uri image-local-url}])]
-      [quo/activity-logs-photos {:photos photos}])
+    (let [images           (or albumMessages message)
+          image-urls       (if albumMessages
+                             (mapv #(get % :image) images)
+                             [(get-in message [:content :image])])
+          image-local-urls (mapv (fn [url]
+                                   {:uri (url/replace-port url (rf/sub [:mediaserver/port]))})
+                                 image-urls)
+          photos           (when image-local-urls image-local-urls)]
+      [quo/activity-logs-photos
+       {:photos photos
```

Comment:
I believe we can pass `image-local-urls` directly here. WYDT?

Reply:
Yup